doc/man3/EVP_EncryptInit.pod

```diff
@@ -16,22 +16,49 @@ EVP_CIPHER_CTX_nid, EVP_CIPHER_CTX_block_size, EVP_CIPHER_CTX_key_length,
+EVP_CIPHER_CTX_set_padding, EVP_enc_null,
+EVP_des_cbc, EVP_des_ecb, EVP_des_cfb, EVP_des_cfb1, EVP_des_cfb8, EVP_des_ofb,
+EVP_des_ede_cbc, EVP_des_ede, EVP_des_ede_ofb, EVP_des_ede_cfb,
+EVP_des_ede3_cbc, EVP_des_ede3, EVP_des_ede3_ofb, EVP_des_ede3_cfb,
+EVP_des_ede3_cfb1, EVP_des_ede3_cfb8, EVP_des_ede3_wrap, EVP_desx_cbc,
+EVP_rc4, EVP_rc4_40, EVP_rc4_hmac_md5,
 EVP_idea_cbc, EVP_idea_ecb, EVP_idea_cfb, EVP_idea_ofb, EVP_rc2_cbc,
 EVP_rc2_ecb, EVP_rc2_cfb, EVP_rc2_ofb, EVP_rc2_40_cbc, EVP_rc2_64_cbc,
-EVP_bf_cbc, EVP_bf_ecb, EVP_bf_cfb, EVP_bf_ofb, EVP_cast5_cbc,
-EVP_cast5_ecb, EVP_cast5_cfb, EVP_cast5_ofb, EVP_rc5_32_12_16_cbc,
-EVP_rc5_32_12_16_ecb, EVP_rc5_32_12_16_cfb, EVP_rc5_32_12_16_ofb,
-EVP_aes_128_cbc, EVP_aes_128_ecb, EVP_aes_128_cfb, EVP_aes_128_ofb,
-EVP_aes_192_cbc, EVP_aes_192_ecb, EVP_aes_192_cfb, EVP_aes_192_ofb,
-EVP_aes_256_cbc, EVP_aes_256_ecb, EVP_aes_256_cfb, EVP_aes_256_ofb,
+EVP_bf_cbc, EVP_bf_ecb, EVP_bf_cfb, EVP_bf_ofb,
+EVP_cast5_cbc, EVP_cast5_ecb, EVP_cast5_cfb, EVP_cast5_ofb,
+EVP_seed_cbc, EVP_seed_ecb, EVP_seed_cfb, EVP_seed_ofb,
+EVP_rc5_32_12_16_cbc, EVP_rc5_32_12_16_ecb, EVP_rc5_32_12_16_cfb, EVP_rc5_32_12_16_ofb,
+EVP_aes_128_cbc, EVP_aes_192_cbc, EVP_aes_256_cbc,
+EVP_aes_128_ecb, EVP_aes_192_ecb, EVP_aes_256_ecb,
+EVP_aes_128_cfb, EVP_aes_192_cfb, EVP_aes_256_cfb,
+EVP_aes_128_cfb1, EVP_aes_192_cfb1, EVP_aes_256_cfb1,
+EVP_aes_128_cfb8, EVP_aes_192_cfb8, EVP_aes_256_cfb8,
+EVP_aes_128_ofb, EVP_aes_192_ofb, EVP_aes_256_ofb,
+EVP_aes_128_ctr, EVP_aes_192_ctr, EVP_aes_256_ctr,
+EVP_aes_128_wrap, EVP_aes_192_wrap, EVP_aes_256_wrap,
+EVP_aes_128_wrap_pad, EVP_aes_192_wrap_pad, EVP_aes_256_wrap_pad,
+EVP_aes_128_xts, EVP_aes_256_xts,
 EVP_aes_128_gcm, EVP_aes_192_gcm, EVP_aes_256_gcm,
+EVP_aes_128_ocb, EVP_aes_192_ocb, EVP_aes_256_ocb,
 EVP_aes_128_ccm, EVP_aes_192_ccm, EVP_aes_256_ccm,
 EVP_aes_128_cbc_hmac_sha1, EVP_aes_256_cbc_hmac_sha1,
 EVP_aes_128_cbc_hmac_sha256, EVP_aes_256_cbc_hmac_sha256,
+EVP_aria_128_cbc, EVP_aria_192_cbc, EVP_aria_256_cbc,
+EVP_aria_128_ecb, EVP_aria_192_ecb, EVP_aria_256_ecb,
+EVP_aria_128_cfb, EVP_aria_192_cfb, EVP_aria_256_cfb,
+EVP_aria_128_cfb1, EVP_aria_192_cfb1, EVP_aria_256_cfb1,
+EVP_aria_128_cfb8, EVP_aria_192_cfb8, EVP_aria_256_cfb8,
+EVP_aria_128_ofb, EVP_aria_192_ofb, EVP_aria_256_ofb,
+EVP_aria_128_ctr, EVP_aria_192_ctr, EVP_aria_256_ctr,
+EVP_aria_128_gcm, EVP_aria_192_gcm, EVP_aria_256_gcm,
+EVP_aria_128_ccm, EVP_aria_192_ccm, EVP_aria_256_ccm,
+EVP_camellia_128_cbc, EVP_camellia_192_cbc, EVP_camellia_256_cbc,
+EVP_camellia_128_ecb, EVP_camellia_192_ecb, EVP_camellia_256_ecb,
+EVP_camellia_128_cfb, EVP_camellia_192_cfb, EVP_camellia_256_cfb,
+EVP_camellia_128_cfb1, EVP_camellia_192_cfb1, EVP_camellia_256_cfb1,
+EVP_camellia_128_cfb8, EVP_camellia_192_cfb8, EVP_camellia_256_cfb8,
+EVP_camellia_128_ofb, EVP_camellia_192_ofb, EVP_camellia_256_ofb,
+EVP_camellia_128_ctr, EVP_camellia_192_ctr, EVP_camellia_256_ctr,
```

Comment:
May I suggest alphabetical order? I for one wouldn't mind the idea of folding key lengths with `{128,192,256}`... Would others?

Reply:
@dot-asm I've followed the document's conventional order here: I believe `cbc` is put first in this document is because it is the default mode. In addition, the header order follows section headers below, which is also quite arbitrary (e.g., AE modes put at the bottom with no separate section heading). Maybe this could be changed too?

All key lengths are now folded in this document. Does it work for you?